lib/controller/matlab/.gitignore

```diff
@@ -277,6 +277,7 @@ wb_supervisor_node_is_proto.m
 wb_supervisor_node_move_viewpoint.m
 wb_supervisor_node_set_visibility.m
 wb_supervisor_node_set_velocity.m
+wb_supervisor_node_set_joint_position.m
```

Comment:
This looks unrelated...

Reply:
It was creating an additional file, Benjamin fixed it today in another PR. After a sync we will not need it any more